core/sup_tree_core/executor_pool/memcache_writer.ex

```diff
@@ -0,0 +1,100 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ExecutorPool.MemcacheWriter do
+  @moduledoc """
+  A `GenServer` to manage memcache for each executor pool.
+  """
+
+  @timeout         300_000
+  @max_records_num 10
```

Comment:
For me `10` seems too few, but I don't have any data how many we should accept...
Can we start with `100` and see what users say about the limit?

Reply:
I didn't consider the number so much.
Ok, I changed the value to 100, in 60b7e30ab96720c852fce613a29e99e0801de475